package.json

```diff
@@ -62,7 +62,7 @@
     "@adobe/helix-shared-wrap": "2.0.2",
     "@adobe/helix-status": "10.1.2",
     "@adobe/helix-universal-logger": "3.0.18",
-    "@adobe/spacecat-shared-data-access": "1.26.1",
+    "@adobe/spacecat-shared-data-access": "1.29.0",
```

Comment:
If you want to test in this PR with the version from your shared PR, you can do it by referencing like described here: https://emmer.dev/blog/installing-npm-packages-from-github/#2.-package.json-is-in-a-subdirectory